tests/pcre/in16.re

```diff
@@ -0,0 +1 @@
+[[:alpha:]]
```

Comment:
```
build/bin/re -r pcre -py tests/pcre/in16.re  > build/tests/pcre/got16.fsm
tests/pcre/in16.re:1: Syntax error: expected group body
```
what's going on here?

Reply:
That's the effect of pre-#75. I'll rebase so the PR head builds.

Reply:
Oh of course. silly me.